### PR TITLE
Overlay the original token logo and symbol for bridged tokens

### PR DESCRIPTION
**Pull request by Lbqds**
The source chain is Alephium and the target chain is Ethereum:
Screenshot 2024-11-06 at 20 37 56

The source chain is Alephium and the target chain is BSC
Screenshot 2024-11-06 at 20 38 03

### CODE REVIEW

File: bridge_ui/src/utils/evm.ts [hunk 1]
```diff
@@ -55,14 +87,15 @@ export async function getEthereumToken(
 }
 
 export async function ethTokenToParsedTokenAccount(
```
Comment (h0ngcha0) [inline bridge_ui/src/utils/evm.ts:89]: [Nit] `evmTokenToParsedTokenAccount`

**Review by h0ngcha0 — APPROVED** [review]
LGTM, only one minor comment.

**Comment by polarker** [comment]
Q: Should we filter the tokens based on the network connected? E.g. If I am connected to Ethereum, then only eth tokens should be selectable

**Comment by Lbqds** [comment]
> Q: Should we filter the tokens based on the network connected? E.g. If I am connected to Ethereum, then only eth tokens should be selectable

Sounds great 👍, will update

**Review by polarker — APPROVED** [review]
LGTM

**Pull request merged**